src/mobot_client/item_drop_session.py

```diff
@@ -1,84 +1,74 @@
 # Copyright (c) 2021 MobileCoin. All rights reserved.
 
 import os
+from decimal import Decimal
+
 import mobilecoin as mc
 import googlemaps
 import pytz
 
+from django.conf import settings
+
 from mobot_client.drop_session import BaseDropSession
 from mobot_client.models import (
     DropSession,
+    Customer,
+    Drop,
     CustomerStorePreferences,
     Order,
     Sku,
-    CustomerDropRefunds, ItemSessionState, OrderStatus,
+    CustomerDropRefunds, SessionState, OrderStatus,
+    Item, OutOfStockException,
 )
 from mobot_client.chat_strings import ChatStrings
 
 
 class ItemDropSession(BaseDropSession):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.gmaps = googlemaps.Client(key=settings.GMAPS_CLIENT_KEY)
+        self.vat_id = settings.VAT_ID
 
-        gmaps_client_key = os.environ["GMAPS_CLIENT_KEY"]
-        self.gmaps = googlemaps.Client(key=gmaps_client_key)
-        
-        self.vat_id = os.environ["VAT_ID"]
-
-    @staticmethod
-    def drop_item_has_stock_remaining(drop):
-        # FIXME: Unused
-        skus = Sku.objects.fliter(item=drop.item)
-        for sku in skus:
-            number_ordered = Order.objects.filter(sku=sku).exclude(status=OrderStatus.CANCELLED).count()
-            if number_ordered < sku.quantity:
-                return True
-
-        return False
-
-    def drop_item_get_available(self, drop_item):
-        available_options = []
-        skus = Sku.objects.filter(item=drop_item).order_by("sort_order")
-
-        for sku in skus:
-            number_ordered = Order.objects.filter(sku=sku).exclude(status=OrderStatus.CANCELLED).count()
-            if number_ordered < sku.quantity:
-                available_options.append(sku)
+    def handle_active_item_drop_session(self, message, drop_session):
```

Comment:
Can we add a docstring describing how we expect the dropsession to behave overall based on different states? It's ok if it's a fairly long docstring - maybe it belongs in the file's docstring as a description of what we want out of this class.